Source/Models/VirtusizeOrderItem.swift

```diff
@@ -24,8 +24,8 @@
 
 /// This structure wraps the parameters of the order item for the API request of sending the order
 public struct VirtusizeOrderItem: Codable {
-    /// The provide ID provided by the client. It must be unique for a product.
-    private let productId: String
+    /// The extenral provide ID provided by the client. It must be unique for a product.
```

Comment:
`extenral` => `external`
```suggestion
    /// The external provide ID provided by the client. It must be unique for a product.
```

Reply:
ok, fixed